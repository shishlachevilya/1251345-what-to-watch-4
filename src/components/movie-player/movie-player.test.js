import React from "react";
import renderer from "react-test-renderer";
import MoviePlayer from "./movie-player";

describe(`MoviePlayer component test`, () => {
  it(`component should renders correctly`, () => {
    const tree = renderer
    .create(
        <MoviePlayer
          videoPrev={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
          poster={`bg-the-grand-budapest-hotel-poster.jpg`}
          isHover={true}
        />
    )
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
