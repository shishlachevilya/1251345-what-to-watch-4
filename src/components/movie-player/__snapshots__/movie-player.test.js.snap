// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlayer component test component should renders correctly 1`] = `
<video
  height="175"
  loop={true}
  muted="off"
  poster="bg-the-grand-budapest-hotel-poster.jpg"
  preload="none"
  width="280"
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
    type="video/mp4"
  />
</video>
`;
